maisim/maisim.Game/Database/BeatmapDatabaseContext.cs

```diff
@@ -16,9 +16,24 @@ public class BeatmapDatabaseContext : DbContext
         public DbSet<BeatmapSet> BeatmapSets { get; set; }
         public string DatabasePath { get; set; }
 
-        public BeatmapDatabaseContext(string databasePath)
+        public BeatmapDatabaseContext()
         {
-            DatabasePath = databasePath;
+
+        }
+
+        public void InitializeDatabase(string databasePath)
```

Comment:
This should initialize the database from the specified `DatabasePath` property of `BeatmapDatabaseContext`